code/modules/power/apc.dm

```diff
@@ -973,6 +973,9 @@
 	if(malf.malfhacking)
 		to_chat(malf, "You are already hacking an APC.")
 		return
+	if(constructed)
+		to_chat(malf, "<span class='warning'>This APC was reciently constructed, and not fully linked to station systems, and is of no use to hack.</span>")
```

Comment:
```suggestion
		to_chat(malf, "<span class='warning'>This APC was only recently constructed, and is not fully linked to station systems. Hacking it would be pointless.</span>")
```